src/utils.hpp

```diff
@@ -0,0 +1,34 @@
+#if 0
+#pragma once
+#include <cstddef>
+#include <string.h>
+
+namespace utils {
+
+	bool safe_strn_copy(char *dest, size_t destSize, const char *src) {
```

Comment:
We could use [`std::char_traits::copy`](https://en.cppreference.com/w/cpp/string/char_traits/copy)

Reply:
Got it, just wrote it as a possible future use in case we didn't want to abuse std, will remove